examples/caldav-get-lib.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+from icalendar import Calendar
```

Comment:
This import can go away (see comments further below) (and hence the "external library" is not so external anymore - the caldav library already has built-in support for icalendar objects!)

Reply:
done